src/components/home/home.module.css

```
@@ -0,0 +1,60 @@
+.content {
+  position: absolute;
+  top: 39%;
+  width: 100%;
+}
```

```suggestion

}
```
all css styles need 1 empty line delimiter. 

do not add it now, but remember for future